src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -245,7 +245,7 @@ public class FunctionInstanceMonitor
 
         public void Start()
         {
-            _logInfo.LogFunctionStart(_invocationId.ToString());
+            _logInfo.LogFunctionStart(_metadata.Name, _invocationId.ToString());
```

Comment:
While at this, can we also log _metadata.IsProxy for both Start and End? There are several asks to log if/when invoking a function to know if it is a proxy or not.

Reply:
FunctionName column will now be populated correctly. Will that be enough?